webapp/src/components/sidebar_right/github_items.tsx

```diff
@@ -185,13 +189,34 @@ function GithubItems(props: GithubItemsProps) {
         if (item.status) {
             switch (item.status) {
             case 'success':
-                status = (<span style={{...style.icon, ...style.iconSucess}}><TickIcon/></span>);
+                status = (
+                    <span
+                        title={'Success'}
```

Comment:
Can this be more descriptive? Maybe use these same wording as GitHub

Reply:
depends on reply to https://github.com/mattermost/mattermost-plugin-github/pull/462#discussion_r747668878

Reply:
I think "Checks Passing" would be more appropriate, but not a required change